manifests/params.pp

```diff
@@ -78,6 +78,7 @@
       $config_group              = 'root'
       $config_owner              = 'root'
       $daemonize                 = true
+      $log_dir_mode              = '0755'
```

Comment:
It's better to move this outside ```case $::osfamily {}```.
There's no point in repeating the same line with the exact same value for each ```$::osfamily```.